File: backend/app/src/main/java/com/smash/domain/group/TimeSlot.java
package com.smash.domain.group;

import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@RequiredArgsConstructor
public enum TimeSlot {
    SLOT_13_15("1-3시"), SLOT_15_17("3-5시");

    private final String label;
}

File: frontend/app/lib/core/domain/time_slot.dart
enum TimeSlot {
  slot1315,
  slot1517;

  String get apiValue => this == TimeSlot.slot1315 ? 'SLOT_13_15' : 'SLOT_15_17';

  static TimeSlot fromApi(String value) => value == 'SLOT_13_15' ? TimeSlot.slot1315 : TimeSlot.slot1517;

  String get koreanLabel => this == TimeSlot.slot1315 ? '1-3시' : '3-5시';
}
